notebooks/Viewing 3D volumetric data with matplotlib.ipynb

```diff
@@ -1081,24 +1140,35 @@
    "cell_type": "code",
    "execution_count": 25,
    "metadata": {
-    "collapsed": true
+    "collapsed": true,
+    "deletable": true,
+    "editable": true
    },
    "outputs": [],
    "source": [
+    "# Read the image \n",
     "struct = nibabel.load(os.path.join(d, 'attention/structural/nsM00587_0002.hdr'))\n",
+    "\n",
+    "# Get a NumPy array\n",
```

Comment:
How about: Get a plain numpy array, without all the metadata

(but maybe that just makes it more complicated, I'm not sure)